content/departments/engineering/dev/process/cloud_processes.md

```diff
@@ -41,3 +41,38 @@ DELETE * FROM orgs WHERE id = $ORG_ID;
 ### Other useful queries
 
 A list of useful DB queries can be found in [useful DB queries](https://docs.google.com/spreadsheets/d/1Z1-7uJwtF2etZFeqTcJS4z9WcAjKlkCxt8HQ931D3dA).
+
+## Closing down an employee's organization on Sourcegraph Cloud
+
+If you have created an organization on Sourcegraph Cloud (in production) for yourself that you’ve been using for testing or learning, follow these instructions when it’s time to delete it:
+
+- Go to your organization settings page. You can access it from the top right menu where there's a link to your org.
+
+- You should see the "Delete this organization" button in the bottom of the page. Go ahead and click it, confirming the deletion by typing the org's name in the input field.
+
+  Note that this action can't be undone and will hard delete the organization along with all resources associated to it, such as:
```

Comment:
```suggestion
  Note that this action can't be undone and will hard delete the organization along with all resources associated to it, including:
```

Non-native speaker, but I think "such as" is often used to make examples, and make me feel that we aren't sure about what will be deleted ourselves.

Reply:
Non-native speaker too. Thanks for suggestion, @unknwon :)